test/unit/rules/no-yield-to-default-test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE = require('../../../lib/rules/no-yield-to-default').ERROR_MESSAGE;
+
+generateRuleTests({
+  name: 'no-yield-to-default',
+
+  config: true,
+
+  good: [
+    '{{yield}}',
+    '{{has-block}}',
+    '{{has-block-params}}',
+    '{{hasBlock}}',
+    '{{hasBlockParams}}',
+  ],
+
+  bad: [
```

Comment:
Can you add more bad examples for (permutations of each of the disallowed keywords):

```hbs
{{#if (has-block "default")}}

{{/if}}
```

```hbs
{{if (has-block "default") "some-value"}}
```

Reply:
done